src/commons/NavigationHeader.tsx

```diff
@@ -348,7 +348,9 @@ const NavigationHeader: FunctionComponent<INavigationHeader> = props => {
               })}
             </TouchableOpacity>
             <View style={styles.smallTitleContainer}>
-              <Text style={styles.smallTitle}>{title}</Text>
+              <Text style={styles.smallTitle}>
+                {title.length > 27 ? title.substring(0, 27) + '...' : title}
```

Comment:
Can't we use ellipsizeMode={'tail'} and give a width in which we'd like the title to fit? There are many different phone and tablet sizes, maybe it's not worth it to only show 27 on tablets where more letters'd fit in one line and maybe there are smaller phones that don't have enough space for 27 chars

Reply:
i thought about it first and tried, but the design specifically says 30 chars and in a way I kinda like it like that.. We could increase it for tablets tho.

Reply:
Please double check with Hamed :)